feat: React migration - PR list CY-6556 (#1831)
codacy/docs: docs/repositories/pull-requests.md

# Quality Pull Requests page

The **Quality Pull Requests page** displays an overview of the pull requests in your repository, such as the analysis status and the code quality metrics for each pull request. This allows you to monitor the code quality of the work in progress in your repository.

> By default, the page lists open pull requests but you can click the **Closed** tab at the top of the list to display the closed pull requests.

Reviewer: A couple of suggestions:
- split this into two sentences 
- invert the second sentence to front-load the result with the structure `To [result], [action].`

```suggestion
By default, the page lists open pull requests. To display closed pull requests, click the **Closed** tab above the list.
```

Reviewer: Let's go with 2 for now 👍

> ![Pull Requests page](images/pull-requests.png)<!--TODO CY-6556 Update screenshot. It might be tricky to update this screenshot only, because all the remaining screenshots for the Pull Request page are related with one-->

Reviewer: Let's sync on this during the daily.

Reviewer 2: This is the open source repository that I used to create the original screenshot:

https://app.codacy.com/gh/pyccel/pyccel/pullRequests

You should be able to grab an updated screenshot from that link.
